OpenTrons/opentrons-api
# fix(app): fix links for setting up new robot

RQA-2844

# Overview

On desktop devices page, clicking 'See how to set up a new robot' opens a modal with a link to support for setting up a new robot. Instead of linking to the OT-2 setup support page, we will now link to the robot's specific quickstart guide.
![image](https://github.com/user-attachments/assets/9f3f3b66-40b2-487b-819b-140a9f6e0bb1)

reference https://github.com/Opentrons/opentrons/pull/15656

# Test Plan

- navigate to desktop app devices page
- click 'See how to set up a new robot' at top right of page
- click link for 'Learn more about setting up a new robot'
- verify that Flex link pushes to https://insights.opentrons.com/hubfs/Products/Flex/Opentrons%20Flex%20Quickstart%20Guide.pdf
- verify that OT-2 link pushes to https://insights.opentrons.com/hubfs/Products/OT-2/OT-2%20Quick%20Start%20Guide.pdf

# Changelog

- update links and copy
- update tests

# Review requests

see test plan

# Risk assessment

low
